packaging/suse/trento-agent.spec

```diff
@@ -29,7 +29,10 @@ Source1:        vendor.tar.gz
 ExclusiveArch:  aarch64 x86_64 ppc64le s390x
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 BuildRequires:  golang(API) = 1.23
+%if 0%{?suse_version} && 0%{?suse_version} < 1600
+# Prometheus Node Exporter is not available in SLES 16
 Requires:       golang-github-prometheus-node_exporter
+%endif
```

Comment:
Can't we just use `Suggests` instead of `Requires` here and get rid of the if block?
Bear in mind we also want to make it optional in SLES 15.